src/windows.toit

```diff
@@ -77,6 +78,28 @@ to other paths.
 is-relative path/string -> bool:
   return not is-rooted path
 
+/**
+Convert the $path to an absolute path by prepending the current working directory to the path
+if it is not already absolute.
+
+The result is cleaned by $clean before being returned.
+*/
+to-absolute path/string -> string:
+  if is-absolute path: return clean path
+  if is-rooted path and is-absolute-volume directory.cwd:
```

Comment:
This is missing support for drive-relative paths, like `C:foo`.
We can only use the cwd if the drive of the rooted path is the same as the one from the current dir. Otherwise the path should be `\`.
For example `C:foo` with a CWD of `D:\foo\bar` should be `C:\foo`.